home-application/pom.xml

```diff
@@ -78,13 +80,30 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>org.springframework.boot</groupId>
-                <artifactId>spring-boot-maven-plugin</artifactId>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>${maven.dependency.plugin.version}</version>
```

Comment:
Please do not add any additional `maven-dependency-plugin` execution. Just rely on the existing.

Reply:
i didn't add any additional plugin, just changed their order

Reply:
this dependency was taken from home-server-generator. It will be removed in the next commit